src/NuGet.Core/NuGet.Packaging/Strings.resx

```diff
@@ -670,4 +669,12 @@ Valid from:</comment>
     <value>The file is not a valid nupkg. File path: {0}</value>
     <comment>0 - Nupkg file path</comment>
   </data>
+  <data name="ArgumentCannotBeNullOrEmpty" xml:space="preserve">
+    <value>The argument cannot be null or empty.</value>
+  </data>
+  <data name="SignError_TimestampResponseUnsupportedDigestAlgorithm" xml:space="preserve">
+    <value>The timestamp response has an unsupported digest algorithm - '{0}'. The following algorithms are supported - '{1}'.</value>
```

Comment:
The timestamp response has an unsupported digest algorithm ({0}). The following algorithms are supported:  {1}.